### PR TITLE
[plugin.video.tagesschau] 2.3.0

### PR DESCRIPTION
**Pull request by christianprasch**
### Description
This is the Tagesschau addon from J.Schumacher and H.Saul updated to Kodi 19 (Matrix)
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by basrieter** [comment]
I would prefer this PR to go via a PR in the original add-on repository of plugin.video.tagesschau: https://github.com/joerns/xbmc-plugin.video.tagesschau/issues.

@joerns Could you let us know if you still accept contributions via your own GitHub or that are OK with us accepting this update version of your add-on?

File: plugin.video.tagesschau/addon.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"></import>
+  </requires>
+  
+  <extension point="xbmc.python.pluginsource" library="main.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <language>de</language>
+    <platform>all</platform>
```
Comment (basrieter) [inline plugin.video.tagesschau/addon.xml:16]: Please add a proper license tag. Example: <license>GPL-3.0-only</license>

Comment (basrieter) [reply plugin.video.tagesschau/addon.xml:16]: Also add a proper <source></source> tag that points to the correct github.

File: plugin.video.tagesschau/addon.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <disclaimer lang="en">Videos from tagesschau.de</disclaimer>
+    <assets>
+      <icon>resources/assets/icon.png</icon>
+      <fanart>resources/assets/fanart.jpg</fanart>
+      <screenshot>resources/assets/screenshot_1.png</screenshot>
+      <screenshot>resources/assets/screenshot_2.png</screenshot>
+      <screenshot>resources/assets/screenshot_3.png</screenshot>
+    </assets>
+    <news>
+version 2.3.0
```
Comment (basrieter) [inline plugin.video.tagesschau/addon.xml:34]: Try to keep this a bit more to the most recent ones.

File: plugin.video.tagesschau/libs/tagesschau_json_api.py [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+        """Returns the parsed JSON structure for the latest broadcasts."""        
+        handle = urllib.request.urlopen("http://www.tagesschau.de/api/multimedia/sendung/letztesendungen100.json")
+        return json.loads(handle.read())
+
+    def archived_broadcasts(self):
+        """Returns the parsed JSON structure for the archived broadcasts."""        
+        handle = urllib.request.urlopen("http://www.tagesschau.de/api/multimedia/sendung/letztesendungen100~_week-true.json")
+        return json.loads(handle.read())
+        
+if __name__ == "__main__":
```
Comment (basrieter) [inline plugin.video.tagesschau/libs/tagesschau_json_api.py]: is this needed?

**Review by basrieter — CHANGES_REQUESTED** [review]
Please have a quick look at my suggestions.

**Comment by joerns** [comment]
> I would prefer this PR to go via a PR in the original add-on repository of plugin.video.tagesschau: https://github.com/joerns/xbmc-plugin.video.tagesschau/issues.
> 
> @joerns Could you let us know if you still accept contributions via your own GitHub or that are OK with us accepting this update version of your add-on?

Hi @basrieter, thanks for checking. @christianprasch did check with me beforehand. I am no longer supporting this plugin, I would be happy if Christian takes over. PRs should then go via his forked repo. Cheers

**Comment by christianprasch** [comment]
Hi @basrieter, i have implemented the requested changes and updated the PR as described in "HOW-TO:Update a pull request to an official addon repository on GitHub". Hopefully i have done it in the correct way, i am completely new to this workflow.
Please let me know when i missed something.
Regards, Christian

**Review by basrieter — APPROVED** [review]

**Comment by basrieter** [comment]
All merged. Thanks for your contribution.

**Pull request merged**